dynamodump/dynamodump.py

```diff
@@ -810,6 +825,10 @@ def do_restore(dynamo, sleep_interval, source_table, destination_table, write_ca
         "WriteCapacityUnits": int(write_capacity),
     }
 
+    optional_args = {}
+    if billing_mode == "PROVISIONED":
```

Comment:
Same here

Reply:
I'm not sure about that but i commited a new proposition, let me know what do you think about